projects/cashmere/src/lib/sass/select.scss

```diff
@@ -56,15 +56,17 @@
     -webkit-appearance: none;
     -moz-appearance: none;
     background-color: $white;
-    border: 1.5px solid $gray-300;
+    border: 1px solid $gray-300;
+    /* box-shadow used to add 0.5px to the border */
+    box-shadow: inset 0.5px 0.5px 0px $gray-300, inset -0.5px -0.5px 0 $gray-300;
     border-radius: 0;
     color: $text;
     cursor: pointer;
     // Font needs to be inherited, because by default <select> uses a system font
     font: inherit;
     height: 35px;
     line-height: 1.3; // avoid clipping character descenders in some browsers
-    padding-left: 7px;
+    padding-left: 9px;
```

Comment:
@corykon - this one is dumb.  On every single browser, the about of padding left added to an option in a select element is different (even with appearance: none).  In Chrome there is none, in Firefox there is a lot, and IE/Safari somewhere in between.  This previous value was based on Firefox, and I realize in Chrome it looked off.  So I'm trying to split the difference here and bump it up slightly.